tests/cross/src/test/scala/tests/pc/CompletionOverrideSuite.scala

```diff
@@ -648,10 +648,60 @@ object CompletionOverrideSuite extends BaseCompletionSuite {
        |}
        |""".stripMargin,
     // NOTE(olafur) assert completion items are empty because it's not possible to
-    // override vars.
+    // override concrete vars.
     ""
   )
 
+  check(
+    "abstract-var",
+    """|package m
+       |abstract class Val {
+       |  var hello1: Int
+       |}
+       |class Main extends Val {
+       |  override var hello@@
+       }
+       |""".stripMargin,
+    "var hello1: Int",
+    includeDetail = false,
+    filterText = "override var hello1"
+  )
+
+  check(
+    "abstract-var-val",
+    """|package m
+       |abstract class Val {
+       |  var hello1: Int
+       |  val hello2: Int
+       |  def hello3: Int
+       |}
+       |class Main extends Val {
+       |  override val hello@@
+       }
+       |""".stripMargin,
+    // NOTE(gabro): we inlcude also var and def completions despite the user typed a val
```

Comment:
I honestly couldn't make this work, but it's not too terrible as UX

Reply:
I think this UX is totally fine, you don't always know if the supermethod is a def, val or a var